versions/3.0.md

```diff
@@ -1848,6 +1920,84 @@ breed: Mixed
 
 ```
 
+#### <a name="describedExampleObject"></a>Described Example Object
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="describedExampleSummary"></a>summary | `string` | Short description for the example.
+<a name="describedExampleDescription"></a>description | `string` | Long description for the example.
+<a name="describedExampleValue"></a>value | [Example Object](#exampleObject) | Embedded literal example. The `value` field and `valueUrl` field are mutually exclusive.
+<a name="describedExampleValueUrl"></a>valueUrl | `string` | A URL that points to the literal example. This provides the ability to reference examples that cannot easily be included in JSON or YAML documents.  The `value` field and `valueUrl` field are mutually exclusive. 
```

Comment:
I suggest using `$ref : uri` instead of `valueUri`.
This way, it can also be a pointer to a value elsewhere in the OpenAPI definition, or it can be an external resource